packages/core/content-manager/server/src/controllers/relations.ts

```diff
@@ -328,8 +328,9 @@ export default {
       }
 
       // Add the status and locale filters if they are provided
-      if (status) {
-        where[`${alias}.published_at`] = getPublishedAtClause(status, targetUid);
+      const publishedAt = getPublishedAtClause(status, targetUid);
```

Comment:
so published -> published / draft -> draft

`getPublishedAtClause` always returns an object so the condition will always be truthy is that a pbl ? Sounds a bit odd to allow that and let the db layer skip it 🤔 

```
where: {
  alias.published_at: {}
}
```

Reply:
good point! will set it so status has to a filled object